app/src/main/java/com/github/gogiffy/fragments/GifGridFragment.java

```diff
@@ -88,6 +88,11 @@ public void onLoadMore(int page, int totalItemsCount) {
         loadTrendingGifs(0);
     }
 
+    private void unUsedMethod(String args){
+        String LONG_LONG_INVALID_STRING="NOT SO FORMATED CODE";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "LONG_LONG_INVALID_STRING". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "LONG_LONG_INVALID_STRING" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00117)